CHANGELOG.md

```diff
@@ -1,5 +1,58 @@
 # Changelog
 
+## [1.58](https://github.com/brave/brave-ios/releases/tag/v1.58)
+
+### Web3
+
+- Added Coinbase Pay as an On-ramp provider. ([#7880](https://github.com/brave/brave-ios/issues/7880))
+- Added Stripe as an On-ramp provider. ([#7794](https://github.com/brave/brave-ios/issues/7794))
+- Added Filecoin support in Brave Wallet. ([#7809](https://github.com/brave/brave-ios/issues/7809))
+- Added ability to choose Solana account when connecting to Solana DApp. ([#7637](https://github.com/brave/brave-ios/issues/7637))
+- Removed placeholder value in "Buy" screen and updated default currency to USD. ([#7709](https://github.com/brave/brave-ios/issues/7709))
+- Fixed wallet backup flow when initiated from "Accounts" tab. ([#8120](https://github.com/brave/brave-ios/issues/8120))
+- Fixed Solana DApps to disconnect when permission is revoked. ([#7688](https://github.com/brave/brave-ios/issues/7688))
+
+### General 
+
+- Implemented “Copy clean URL” feature on iOS. ([#6179](https://github.com/brave/brave-ios/issues/6179))
+- Added support for Apple Screentime. ([#3939](https://github.com/brave/brave-ios/issues/3939))
+- Added support for more regions for Brave News opt-in card. ([#7932](https://github.com/brave/brave-ios/issues/7932))
```

Comment:
I think we can just say that we added Brave News support for those specific regions. For example:

```
- Added Brave News support for "Germany", "France", "India" & "Austria".
```